spring-cloud-gcp-data-spanner/src/main/java/org/springframework/cloud/gcp/data/spanner/repository/query/SqlSpannerQuery.java

```diff
@@ -199,13 +199,12 @@ public List executeRawResult(Object[] parameters) {
 		Sort sort = null;
 
 		for (Object param : parameters) {
-			Class<?> paramClass = param.getClass();
-			if (isPageableOrSort(paramClass)) {
+			if (param != null && isPageableOrSort(param.getClass())) {
```

Comment:
what if `param` is `null`, but also a `Pageable` or `Sort`?

Reply:
NPE would be thrown I guess?
Is this PR changing that behavior?

Reply:
Yeah I think that is outside the scope of this PR; here I am just addressing passing in `null` as a param in a parameterized query.